shark/src/main/java/shark/internal/Strings.kt

```diff
@@ -26,14 +23,8 @@ private fun createHash(
     // Create MD5 Hash.
     val digest = MessageDigest.getInstance(algorithm)
     digest.update(text.getBytes())
-    val messageDigest = digest.digest()
 
-    // Create Hex String.
-    val hexString = StringBuilder()
-    for (b in messageDigest) {
-      hexString.append(Integer.toHexString(0xff and b.toInt()))
-    }
-    return hexString.toString()
+    return BigInteger(1, digest.digest()).toString(16).padStart(40, '0')
```

Comment:
Thx! We should actually just remove this method alltogether and use `text.encodeUtf8().sha1().hex()` instead

Reply:
Cool, thanks! Updated.